src/bees.cc

```diff
@@ -693,9 +693,14 @@ bees_main(int argc, char *argv[])
 				break;
 
 			case 'h':
-			default:
 				do_cmd_help(argv);
-				return EXIT_FAILURE;
+				return EXIT_SUCCESS;
+			default:
```

Comment:
Add an empty line before "default" to match the other cases.

Reply:
I think I got confused by how Github shows the diffs... There are actually no empty lines. If you inserted them, you can revert that. Sorry.

Reply:
Done: https://github.com/Zygo/bees/pull/277/commits/1b8c0173f1e51e3437e8bd2372c390effef2b0a4